packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -636,3 +654,85 @@ export type AudioTrack = {
+ *
+ * > If you are trying to optimize for scrubbing (many frequent seeks), also see [`ScrubbingModeOptions`](#scrubbingmodeoptions-1).
+ *
+ * @platform android
+ * @paramrm ios
+ */
+export type SeekTolerance = {
+  /**
+   * The maximum time that the actual position seeked to may precede the requested seek position, in seconds. Must be non-negative.
+   * @default 0
+   */
+  toleranceBefore?: number;
+
+  /**
+   * The maximum time that the actual position seeked to may exceed the requested seek position, in seconds. Must be non-negative.
+   * @default 0
+   */
+  toleranceAfter?: number;
+};
+
+/**
+ * Defines scrubbing mode options used by a [`VideoPlayer`](#videoplayer).
+ */
+export type ScrubbingModeOptions = {
+  /**
+   * Sets whether to optimize the player for scrubbing (many frequent seeks).
+   *
+   * You should only enable this, when the player is receiving a large number of seeks in a short period of time. For less frequent seeks, fine-tuning the [`SeekTolerance`](#seektolerance) may be sufficient.
+   *
+   * On Android, the player may consume more resources in this mode, so it should only be used for short periods of time in response to user interaction (e.g. dragging on a progress bar UI element).
+   *
+   * On Android, when scrubbingModeEnabled is true, the playback is suppressed. You should set this property back to `false` when the user interaction ends to allow the playback to resume.
+   * For best results, on iOS you should pause the playback when scrubbing.
+   *
+   * > For best scrubbing performance, consider also increasing the seeking tolerance using the [`SeekTolerance`](#seektolerance-1) property.
+   *
+   * > Other scrubbing mode options will have no effect when this is `false`.
+   * @default false
+   * @platform android
+   * @platform ios
+   */
+  scrubbingModeEnabled?: boolean;
+
+  /**
+   * Whether the codec operating rate should be increased in scrubbing mode
+   *
+   * @platform android
+   * @default true
+   */
+  increaseCodecOperatingRate?: boolean;
+
+  /**
+   * Sets whether ExoPlayer's dynamic scheduling should be enabled in scrubbing mode.
+   * This can result in available output buffers being handled more quickly when seeking.
+   *
+   * @platform android
+   * @default true
+   */
+  enableDynamicScheduling?: boolean;
+  /**
+   * Sets whether to use MediaCodec.BUFFER_FLAG_DECODE_ONLY in scrubbing mode.
+   * When playback is using MediaCodec on API 34+, this flag can speed up seeking by signalling that the decoded output of buffers between the previous keyframe and the target frame is not needed by the player.
+   *
+   * @platform android
+   * @default true
+   */
+  useDecodeOnlyFlag?: boolean;
+
+  /**
+   * Sets whether to avoid flushing the decoder (where possible) in scrubbing mode.
+   * Setting this to true will avoid flushing the decoder when a new seek starts decoding from a key-frame in compatible content.
```

Comment:
```suggestion
   * By default, it is set to `true` and avoids flushing the decoder when a new seek starts decoding from a key-frame in compatible content.
```

If the default value is already, then using the property would automatically avoid flushing.